spec/models/gear_spec.rb

```diff
@@ -14,5 +14,11 @@
 require 'rails_helper'
 
 RSpec.describe Gear, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { is_expected.to validate_presence_of(:name) }
```

Comment:
Would you mind removing this as well? Tests like these are only testing that Rails +shoulda matcher is working the way it should, and that the model is configured the same way as the test. This slows down the test run without providing much value.